### ISSUE TITLE
Integrate automated tests with building nightly

### ISSUE THREAD

**Issue by bajiat** [apinf/platform#1712]

**Comment by bajiat** [comment]
@shaliko Could you add a testing step to travis.yml? We expect the tests to fail.

**Comment by shaliko** [comment]
@bajiat Looks like platform dose not have any tests (Unit, Integrations/Regression).

We can easily update travis.yml for run tests, when we will have tests.

**Comment by bajiat** [comment]
@shaliko Can you take a look at this repo: https://github.com/apinf/apinf-testing ?

**Comment by shaliko** [comment]
Require some time for tests. By quick glance, probably we can use Chrome Headless https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md for run chrome without UI part (ping @brylie).

**Comment by brylie** [comment]
Sounds good. The tests are written to use Selenium, if that has any effect on your decision.

**Comment by shaliko** [comment]
Yes, Selenium should easy work with Chrome Headless http://www.chrisle.me/2013/08/running-headless-selenium-with-chrome/

But that is not final decision, I will test it on Travis CI. More concern how to easy debug if tests fails.

**Comment by shaliko** [comment]
@brylie sorry, above was wrong article "headless Selenium" instead of "headless chrome".

That one right https://blog.codeship.com/laravel-and-behat-using-selenium-and-headless-chrome/

**Comment by shaliko** [comment]
@bajiat Will be good start when all existing test will be fixed.

**Comment by ilarimikkonen** [comment]
after next
